python/apps/taiga/src/taiga/workflows/api/__init__.py

```diff
@@ -241,6 +241,6 @@ async def get_workflow_status_or_404(project_id: UUID, workflow_slug: str, id: U
         project_id=project_id, workflow_slug=workflow_slug, id=id
     )
     if workflow_status is None:
-        raise ex.NotFoundError(f"Workflow status {id} does not exist")
+        raise ex.NotFoundError("Workflow status does not exist")
```

Comment:
Idem

Reply:
I can't return any related b64ID from the internall managed uuids.